test/testsets.txt

```diff
@@ -26,7 +26,7 @@ component/ota/*.test.js
 component/turen/*.test.js
 descriptor/*.test.js
 descriptor/tts/*.test.js
-runtime/services/lightd/*.test.js
-runtime/services/lightd/effects/*.test.js
 runtime/cloudapi-util.*.js
 runtime/cloudapi-*.test.js
+services/lightd/**/*.test.js
+services/ttsd/**/*.test.js
```

Comment:
`services/**/*.test.js`? The multimediad and otad are missing.

Reply:
Yes, these two are not valid tests. I did not put those to the list by intent.